### PR TITLE
Fixes #254, #262, #267, #269

### LINKED ISSUE #254: Errors are continually added when attempting to log in
They eventually go away but the timeout is super high and are not cleared when you navigate between sign in pages in the upload nexus and there is that initial bullet point which is strange formatting.

Screen Shot 2020-06-25 at 09 40 27

### LINKED ISSUE #267: Errors are not dismissed automatically
You can see here that I've caused a few errors and navigated around.

All the errors stick around until hitting dismiss and you can just keep adding and adding them.

Screen Shot 2020-06-25 at 14 29 28

### LINKED ISSUE #268: No unified error look and feel
You can see there are two different ways to display an error...
Screen Shot 2020-06-25 at 14 29 28

Screen Shot 2020-06-25 at 09 40 27

### LINKED ISSUE #269: Spacing issue on error popup options
You can see here that there is a whole lot of space between the two actions I can take in response to this error.

Screen Shot 2020-06-25 at 14 29 28

### PR DESCRIPTION
**Pull request by robthew**
- [x] Unified Errors

- [x] Better timeouts

- [x] Error spacing Issue

- [x] Errors dismiss after changing tabs within the upload nexus

- [x] Rebase against master

- [x] Fix sign off issue

### CODE REVIEW

**Review by Bialogs — CHANGES_REQUESTED** [review]
I think that the timeout for the vuetify error (3s) is a bit short to read the full error you get when you upload something and receive an error. Maybe something in the 8-10s range would be appropriate for those longer error messages. The width looks much better now though.
As for the errors on the UploadNexus...Errors are not cleared when navigating between the specific page in the upload nexus you are on which was part of #254. What I mean by that is if I encounter an error when logging in to S3 but then I click on the Splunk tab the errors should be removed.

File: src/plugins/vuetify.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ let options: ToastOptions = {
     }
   ],
   // icon: 'mdi-alert-octagram',
-  duration: 1e12 // effectively infinite.
+  duration: 3000 // effectively infinite.
```
Comment (rbclark) [inline src/plugins/vuetify.ts]: The comment on this is now a bit confusing.

Comment (robthew) [reply src/plugins/vuetify.ts]: Why does the UploadNexus have a different errors implementation than the rest of the system? Is there a reason it is not sending its errors to Toast?

I also am seeing a warning about the errors component:

[Vue warn]: The computed property "errors" is already defined in data.

found in

---> <ErrorTooltip>
       <SplunkReader> at src/components/global/upload_tabs/splunk/SplunkReader.vue
         <VTabItem>
           <VTabsItems>
             <VTabs>
               <ThemeProvider>
                 <VDialog>
                   <Modal> at src/components/global/Modal.vue
                     <UploadNexus> at src/components/global/UploadNexus.vue
                       <Landing> at src/views/Landing.vue
                         <VApp>
                           <App> at src/App.vue
                             <Root> vue.runtime.esm.js:619
    VueJS 17

This shows up for SplunkReader and S3Reader.

Comment (Bialogs) [reply src/plugins/vuetify.ts]: Not sure...if you want to unify errors that would be good with me.

Comment (rbclark) [reply src/plugins/vuetify.ts]: I'm not sure who implemented that however no component should have its own error implementation, unifying that was the intended purpose of #268.

**Comment by Bialogs** [comment]
I placed a checklist in the PR description to track what I'd like to see done before this is merged. I think it looks good except for the one left blank.

**Comment by robthew** [comment]
The ErrorTooltip has been replaced with the Toast error popups which dismiss automatically. This should take care of the changing tabs issue.

What is the sign off issue that needs fixing?

**Comment by Bialogs** [comment]
> What is the sign off issue that needs fixing?

The contributor agreement as part of the automatic checks. You can see that under the automated checks section of the PR.

**Comment by Bialogs** [comment]
Also seeing the following issue on this build of heimdall-lite which is not present on netlify...Please note the search bar text as well as the fact I can access this part of the application with an invalid & malformed login. 

Looks like something changed and the error isn't blocking the next action from occurring.

output

File: src/sass/main.scss [hunk 2]
```diff
@@ -87,3 +87,8 @@ $code-kbd-font-weight: false;
 .show {
   display: block !important;
 }
+
+.nobull {
+  list-style-type: none;
+ }
```
Comment (rbclark) [inline src/sass/main.scss]: Is this used anywhere?

Comment (robthew) [reply src/sass/main.scss]: It was used in the ErrorTooltip component, but that has since been removed

Comment (Bialogs) [reply src/sass/main.scss]: Let's remove the dead code.

**Review by Bialogs — CHANGES_REQUESTED** [review]
Errors are still sticking around when navigating between tabs in the upload nexus and you can still bypass the changes you made to the Splunk login by hitting the login button twice in a row.

**Comment by Bialogs** [comment]
I think the best way to fix the login to Splunk issue is to perform additional validation on the hostname the user is trying to connect to.

**Review by Bialogs — DISMISSED** [review]

**Review by Bialogs — APPROVED** [review]

**Pull request merged**